lib/decanter/configuration.rb

```diff
@@ -4,7 +4,7 @@ class Configuration
     attr_accessor :strict
 
     def initialize
-      @strict = :with_exception
+      @strict = true
```

Comment:
does the readme need to be updated or are you doing all of that in a later PR?

Reply:
I actually did it in a previous pr 😳 but I'm going to open a PR with a migration guide soon.

Reply:
ah here: https://github.com/LaunchPadLab/decanter/pull/89/files